libs/core/algorithms/include/hpx/parallel/algorithms/uninitialized_move.hpp

```diff
@@ -286,21 +260,21 @@ namespace hpx::parallel {
             using zip_iterator = hpx::util::zip_iterator<Iter, FwdIter2>;
             using partition_result_type = std::pair<FwdIter2, FwdIter2>;
 
-            util::cancellation_token<util::detail::no_data> tok;
             return util::partitioner_with_cleanup<ExPolicy,
                 util::in_out_result<Iter, FwdIter2>, partition_result_type>::
                 call(
                     HPX_FORWARD(ExPolicy, policy), zip_iterator(first, dest),
                     count,
-                    [tok](zip_iterator t, std::size_t part_size) mutable
+                    [&policy](zip_iterator t, std::size_t part_size) mutable
```

Comment:
In other places you captured `policy` by value. Is there a rationale for capturing by reference here?

Reply:
will fix the inconsistency, capturing by reference everywhere is the better choice right?

Reply:
I think capturing by value is the safest option, especially for the asynchronous variations.